src/static/js/webmentions.js

```diff
@@ -0,0 +1,209 @@
+// Code related to parsing and showing webmentions
+
+// Gets the webmentions json data from current URL
+async function getWebmentions(targetURL) {
+  const apiURL = `https://webmention.io/api/mentions.jf2?perPage=500&target=${targetURL}`;
+  let mentions = []
```

Comment:
Fix code scanning issue:

```suggestion
  let mentions = [];
```